Okta/OktaAuth/OktaApi.swift

```diff
@@ -10,51 +10,82 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-class OktaApi: NSObject {
-
+protocol OktaApi {
```

Comment:
Do we really need this protocol? It is used only in this file
Also not a great name for protocol. Probably we can change to something like:
```suggestion
protocol OktaHttpProtocol {
```